core/controlplane/config/templates/cluster.yaml

```diff
@@ -387,6 +387,16 @@ worker:
 #      # Documentation: http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/dedicated-instance.html
 #      tenancy: default
 #
+#      # (Experimental) GPU Driver installation support
+#      # Currently, only Nvidia driver is supported.
+#      # This setting takes effect only when node's instance family is p2 of g2.
+#      # Otherwise, installation will be skipped even if enabled.
```

Comment:
oh, ok. yes, I agree with you.  kube-aws should prohibit 'enabled:true' with instance types which doesn't support GPU.  I will update my code.